vectordatasource/transform.py

```diff
@@ -592,14 +592,18 @@ def _normalize_country_code(x):
 
 osm_l10n_lookup = set([
     'zh-min-nan',
-    'zh-yue',
 ])
 
 # key is the name in OSM source; value is the Tilezen internal name
```

Comment:
Update the comment here. Value is a tuple where the second field is a priority?

Reply:
> Update the comment here. Value is a tuple where the second field is a priority?

good point, will update in the next commit